### PR TITLE
Users/vedgarbha/paportal actions update

### PR DESCRIPTION
**Pull request by vedg**
Added model version parameter to Upload and Download paportal action

Testing:

gulp ci

### CODE REVIEW

File: src/tasks/download-paportal/download-paportal-v2/index.ts [hunk 1]
```diff
@@ -34,6 +34,7 @@ export async function main(): Promise<void> {
       websiteId: parameterMap["WebsiteId"],
       overwrite: parameterMap["Overwrite"],
       excludeEntities: parameterMap["ExcludeEntities"],
+      modelVersion: parameterMap["ModelVersion"],
```
Comment (tehcrashxor) [inline src/tasks/download-paportal/download-paportal-v2/index.ts:37]: You'll also need to add the parameter to the `tasks.json` in the same directory.

File: src/tasks/upload-paportal/upload-paportal-v2/index.ts [hunk 2]
```diff
@@ -29,5 +29,6 @@ export async function main(): Promise<void> {
     environmentUrl: getEnvironmentUrl(),
     path: parameterMap['UploadPath'],
     deploymentProfile: parameterMap['DeploymentProfile'],
+    modelVersion : parameterMap['ModelVersion'],
```
Comment (tehcrashxor) [inline src/tasks/upload-paportal/upload-paportal-v2/index.ts:32]: You'll also need to add the parameter to the `tasks.json` in the same directory.

Comment (vedg) [reply src/tasks/upload-paportal/upload-paportal-v2/index.ts:32]: Added

File: src/tasks/download-paportal/download-paportal-v2/index.ts (hunk 1 above)
Comment (vedg) [reply src/tasks/download-paportal/download-paportal-v2/index.ts:37]: Added

**Review by tehcrashxor — APPROVED** [review]

**Pull request merged**